src/metabase/util/malli/registry.cljc

```diff
@@ -2,11 +2,12 @@
   (:refer-clojure :exclude [def])
   (:require
    [malli.core :as mc]
-   [malli.registry :as mr])
+   [malli.registry :as mr]
+   [malli.util :as mut])
   #?(:cljs (:require-macros [metabase.util.malli.registry])))
 
 (defonce ^:private registry*
-  (atom (mc/default-schemas)))
+  (atom (merge (mc/default-schemas) (mut/schemas))))
```

Comment:
`(mut/schemas)` add support for declarative merge, untion, etc.

Reply:
Fancy!